### ISSUE TITLE
Different realms for aerosol free radiation fluxes

### ISSUE THREAD

**Issue by oligan** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#257]
rlutaf/rlutcsaf/rsutaf/rsutcsaf are in the aerosol realm while rsdcaf/rsucsaf/rsdscsaf/rsuscsaf are in the atmos realm. It makes more sense to have everyone in atmos. 

rsdscsaf and rsuscsaf duplicates the lowermost layer of rsdcsaf and rsucsaf (but could be useful for those groups not providing the 3D fields). 

I would make sense to request rsdsaf/rsusaf/rldsaf/rlusaf as well (same logics on the naming) as well as the LW clear-sky surface fluxes (rldscsaf/rluscsaf)

**Comment by pierrenabat** [comment]
I agree with you, these variables should be in the atmos realm.
I have another question on these variables : do we have to remove all the aerosols (tropospheric and stratospheric) to calculate these fluxes, or only the tropospheric aerosols ?
Maybe this could be clarified in the data request.

**Comment by martinjuckes** [comment]
I've modified rsucsafbnd,rsucsaf, rsdcsafbnd, rsdcsaf, rsdscsafbnd, rsdscsaf, rsuscsafbnd, rsuscsaf to realm aerosol;

It is too late to add new variables to the request now. Often potentially useful variables are omitted to avoid putting excessive strain on the modeling centres. 

The clear sky fluxes have the description "Calculated in the absence of aerosols and clouds (following Ghan)" which we arrived at after some discussion. Ghan (ACP, 2013) refers to "neglecting  the  scattering  and  absorption  of  solar  radiation  by  all  of  the  aerosol". I could change the description to "... absence of all aerosols ..." and add the DOI for the Ghan paper (doi:10.5194/acp-13-9971-2013 ).
